### DISCUSSION TITLE
Endoint Protection

### DISCUSSION THREAD

**Discussion by tihobanov** [SamWSoftware/serverless-auto-swagger#91] [Q&A] [answered]
Since I need to have my swagger documentation on UAT/Dev Environment, is there a way to protect the swagger endpoints behind some Authorisation method (token, user/pass)?

**Comment by bfaulk96** [comment, marked as answer]
This feature has been added as of release `v2.9.2` 😄 

You can add a custom authorizer for `/swagger` and `/swagger.json` with the custom config option

```yaml
lambdaAuthorizer: ${self:custom.customAuthorizer}
```
